api/proto/config/storage.proto

```diff
@@ -34,13 +35,17 @@ message DatastoreConfig {
 
 	string apiKey = 3;
 
-	string password = 4;
+    // to be decrypted, use cipherTextPassword instead
+    string password = 4;
 
 	// depending on datastore types, it could be bucket or path
 	string dpath = 5;
 
 	// Applies for some datastore types
 	string region = 6;
+
+    // contains the cipher payload
```

Comment:
Specify which fields e.g., "the embedded identity replaces the apiKey and the embedded password replaces the password above"

Reply:
will do